portfolio/src/main/webapp/script.js

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function getList() {
-    fetch("/data").then(response => response.json()).then((list) => {
-        console.log(list);
-        document.getElementById("list-container").innerHTML = list;
+//Fetch list of comments from server and print them on page
+function getComments() {
+    fetch("/data").then(response => response.json()).then((comments) => {
+        document.getElementById("comments-container").innerHTML = comments;
```

Comment:
If you set the innerHTML content directly to `comments`, you will end up with "Pineapple on pizza is great,Dogs are overrated,Alpacas are awesome". Try to manipulate the `comments` object and write a formatted HTML. For example, you can have a `ul` and for each comment you add a `li`.